Makefile

```diff
@@ -58,26 +58,28 @@ test-safety:
 # Creates a release build in a containerized build environment of the static library for Alpine Linux (.a)
 release-build-alpine:
 	rm -rf target/release
-	# build the muslc *.a file
```

Comment:
Alpine uses muslc as it is, so how about leave this comment? 'build the muslc'
I think it seems easy to understand why Alpine has a separate build logic just by looking at the makefile.

Reply:
Restored the comments as your opinion.